### PR TITLE
Update Libraries

### PR DESCRIPTION
**Pull request by wangmengyan95**
1. Update FB Utils to open source 1.10.3
2. Update Twitter Utils to open source 1.10.3
3. Update Parse to 1.10.3
4. Update FB to 4.6.0
5. Update Android Support Library to 23.0.1 
6. Delete FB Utils and Twitter Utils jar
7. Update README.MD

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,13 +18,13 @@ ext {
     compileSdkVersion = 21
     buildToolsVersion = "21.1.1"
 
-    minSdkVersion = 9
+    minSdkVersion = 15
```
Comment (grantland) [inline build.gradle]: what's the reason for this change?

Comment (wangmengyan95) [inline build.gradle]: FB SDK requires this.

Comment (grantland) [inline build.gradle]: Is this variable only used for the samples? It looks like ParseUI-Login and ParseUI-Widget still have `minSdkVersion = 9`. If this is the case, I think it's fine.

**Comment by grantland** [comment]
LGTM, we'll be targeting FB SDK 4.4.0 for minSdkVersion 9 compatibility, but 4.6.0 for our samples.
